packages/web-components/@groupby/elements-search-box/stories/index.ts

```diff
@@ -11,6 +17,14 @@ const updateTextEvent = [
   },
 ];
 
+addStorybookListeners([
+  SEARCHBOX_CLEAR,
+  SEARCHBOX_CLICK,
+  SEARCHBOX_INPUT,
+  SEARCH_REQUEST,
+  UPDATE_SEARCH_TERM
```

Comment:
Please use trailing commas.
```suggestion
  UPDATE_SEARCH_TERM,
```